frontend/src/components/UpgradeImage.tsx

```diff
@@ -0,0 +1,11 @@
+import { imageType, IUpgradeImage } from "../../../shared/types";
+
+export default function UpgradeImage({ picture }) {
```

Comment:
You can utilize the `IUpgradeImage` type for `picture` - the standard React typescript way is to create an `IUpgradeImageProps` interface that has a `picture: IUpgradeImage` property.